### DISCUSSION TITLE
Service startup problem

### DISCUSSION THREAD

**Discussion by Mhalkyo** [rustdesk/rustdesk#9305] [General]
### Bug Description

Rustdesk is installed on my computer and when I start the computer, from time to time the rustdesk service does not start.

### How to Reproduce

I have no idea how this error occurs because when I restart the computer the service starts correctly

### Expected Behavior

I would like there to be no problem starting the service, as this can be problematic on servers if we need to take over.

### Operating system(s) on local side and remote side

Windows 11

### RustDesk Version(s) on local side and remote side

1.3.0

### Screenshots

I don't have a screenshot.

**Comment by rustdesk** [comment]
Is there anything here, `C:\Windows\ServiceProfiles\LocalService\AppData\Roaming\RustDesk\log\server\`
and `C:\Windows\ServiceProfiles\LocalService\AppData\Roaming\RustDesk\log\service\`?

**Reply by Mhalkyo** [reply]
I have several log files for today. I don't remember the exact time.

[service.zip](https://github.com/user-attachments/files/16929117/service.zip)
[server.zip](https://github.com/user-attachments/files/16929120/server.zip)
